docker/gaffer-ui/Dockerfile

```diff
@@ -20,31 +20,22 @@ ARG BASE_IMAGE_TAG=10.1.0.Final
 
 FROM ${BUILDER_IMAGE_NAME}:${BUILDER_IMAGE_TAG} as builder
 
-ARG GAFFER_VERSION=1.12.0
-ARG GAFFER_TOOLS_VERSION=1.12.0
+ARG GAFFER_TOOLS_VERSION=allow-custom-endpoint-path
```

Comment:
Is this intentional?

Reply:
For now yes. This is because I'm building of certain branches. Once I do a 1.15 release with these features, I'll update it